docs/eleventy.config.js

```diff
@@ -31,33 +34,35 @@ module.exports = function (eleventyConfig) {
       },
       copyright: {
         text: '© X-GOVUK'
+      },
+      meta: {
+        items: []
       }
     }
   })
 
-  // Ignores
-  eleventyConfig.ignores.add('**/*/*.js')
-  eleventyConfig.ignores.add('**/*/*.scss')
-  eleventyConfig.ignores.add('**/*/*.njk')
+  eleventyConfig.addNunjucksGlobal("getNunjucksCode", function(componentName) {
+    let data = matter(fs.readFileSync(`examples/${componentName}.njk`, 'utf-8')).content
+
+    data = data.replace(/\{\%\sfrom\s[^\n]+\n\n/, '')
 
-  // Transforms
-  eleventyConfig.addTransform('remove-h1', (content, outputPath) => {
-    // Remove first `h1` as it repeats what’s already shown in page title
-    content = content.replace(/<h1\s*.*tabindex="-1"\s*.*>\s*.*<\/h1>/, '')
-    return content
+    const highlightedCode = hljs.highlight(data, { language: 'js' }).value
```

Comment:
Where are we using this? We already include [the highlighter in `markdown-it-govuk`](https://github.com/x-govuk/markdown-it-govuk#code-highlighting) which is depended on by the Eleventy plugin, so maybe we can use that instead?